syntaxes/omt.tmLanguage.json

```diff
@@ -283,6 +283,28 @@
                 }
             ]
         },
+        "_merge-lists-def": {
+            "patterns": [
+                { "include": "#comments" },
+                {
+                    "begin": "(?:\\G|^) +(subjects|predicates|when): +",
+                    "beginCaptures": {
+                        "1": { "name": "keyword.control.omt" }
+                    },
+                    "end": "$",
+                    "contentName": "meta.embedded.inline.odt",
+                    "patterns": [{ "include": "source.odt" }]
+                },
+                {
+                    "match": "(?:\\G|^) +(from): *(parent|both)?(.*)$",
```

Comment:
The options `parent` and `both` are recognised here, but effectively all `!MergeLists` handlers act as though they're given the option `both`.

Reply:
Implement differences between options. then this can be resolved.

Reply:
This is fixed now, but I guess the CHANGELOG.md date needs an update and then we can merge :)